supriya/realtime/Server.py

```diff
@@ -598,10 +598,13 @@ def boot(
         import supriya.realtime
         if self.is_running:
             return self
-        scsynth_path_candidates = uqbar.io.find_executable('scsynth')
-        if not scsynth_path_candidates:
-            raise RuntimeError('Cannot find scsynth')
-        scsynth_path = scsynth_path_candidates[0]
+        from os import environ
+        scsynth_path = environ.get('SCSYNTH_PATH')
```

Comment:
`os` is already imported at the top of the file, so `os.environ('SCYSNTH_PATH')` is sufficient here.